supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -72,7 +72,12 @@ Warn users about the following deprecation stages:
 * Plan to remove
 * Remove
 
-If available, inform users of alternative capabilities and workarounds.
+Inform users of alternative capabilities and workarounds, if available. 
+
+Avoid predictions about specific releases or release dates. 
+If necessary, you can you can mention current plans to deprecate or remove a capability in a specific release.
```

Comment:
Should we make it clearer that they should still use careful wording? 

Like, should we allow people to say "\<x> will be deprecated in version 4.16."? Or should we have people say something more careful like "\<x> is currently planned to be deprecated in version 4.16." or similar?

I usually push people to use the latter, but not sure what others think.

Not only because there's always a chance that the plans could change, but we're also talking about a future release that likely hasn't been released yet, so that version number could potentially change. (Example: OCP 4.0 was planned, but ended up being skipped in favor of 4.1 at the last second. Same with OCP 3.10 doesn't exist - it went straight from 3.9 to 3.11).

Reply:
Taking a cue from @bergerhoffer's feedback, I've drafted an example below that could serve as inspiration.

```
[[deprecated-functionality]]
=== Deprecated Functionality

Documenting the deprecation stages of software features requires careful and precise communication. Highlight the following stages to users:

* Plan to Deprecate
* Deprecate
* Plan to Remove
* Remove

When alternatives or workarounds are available, clearly inform users. Avoid definitive statements about specific releases or dates for deprecation or removal. If it is necessary to mention a release, use provisional language to reflect the fluid nature of development plans. For example, rather than stating "<x> will be deprecated in version 4.16," use "It is currently planned for <x> to be deprecated in version 4.16." This approach acknowledges the potential for plans to change, as seen in the OpenShift Container Platform's update from the expected version 4.0 to 4.1. By using cautious wording, we set realistic expectations and maintain trust through transparent communication of any potential changes.
```

Reply:
There is already the NOTE in the next section about avoiding specific release numbers.

Reply:
@bergerhoffer @rolfedh @sbmetz - Inspired by your comments, I've reworked this section entirely base on the suggested text from Rolfe. Can you please take another look when you get a chance? Thank you!